rotkehlchen/chain/evm/decoding/cowswap/decoder.py

```diff
@@ -369,7 +371,28 @@ def _decode_normal_claim(self, context: DecoderContext) -> DecodingOutput:
         raw_amount = hex_or_bytes_to_int(context.tx_log.data[128:160])
         amount = asset_normalized_value(amount=raw_amount, asset=self.vcow_token)
         airdrop_identifier: Literal['cow_mainnet', 'cow_gnosis'] = 'cow_mainnet' if self.evm_inquirer.chain_id == ChainID.ETHEREUM else 'cow_gnosis'  # noqa: E501
+        # claimTypes with payment: 1=GnoOption, 2=UserOption, 3=Investor
+        claim_supports_payment = hex_or_bytes_to_int(context.tx_log.data[32:64]) in (1, 2, 3)
+        claimant_address = hex_or_bytes_to_address(context.tx_log.data[64:96])
+        if not self.base.is_tracked(claimant_address):
+            return DEFAULT_DECODING_OUTPUT
+        claim_has_payment = False
```

Comment:
Leave newlines between codeblocks like this for readability

```suggestion
            return DEFAULT_DECODING_OUTPUT

        claim_has_payment = False
```